VTUL/dlp-access
# Social sharing buttons

**JIRA Ticket**: (link) (:star:)
https://webapps.es.vt.edu/jira/browse/LIBTD-2167

# What does this Pull Request do? (:star:)
Adds social media sharing to archive and collection pages. Users can choose social media platforms to display in the admin site.

# What's the changes? (:star:)
-adds react-share package
-adds socialButtons.js component and CSS
-siteForm.js - adds form to let users select social media platforms
-archivePage.js, collectionListView.js, collectionShowPage.js - all include the new socialButtons component
*a couple of other small, unrelated fixes are included in the PR.

# How should this be tested?
In the admin site for any library, choose some social media buttons to include. Then navigate to pages for a collection and an item to see those buttons displayed. If no social media options are selected, then the share section should not display.

# Additional Notes:
Branch is LIBTD-2167

# Interested parties
@yinlinchen 

(:star:) Required fields
